x-pack/snapshot-tool/build.gradle

```diff
@@ -61,32 +74,36 @@ task thirdPartyTest(type: Test) {
   systemProperty 'test.s3.base', s3PermanentBasePath
 }
 
-task writeDockerFile {
-  File minioDockerfile = new File("${project.buildDir}/minio-docker/Dockerfile")
-  outputs.file(minioDockerfile)
-  doLast {
-    minioDockerfile.parentFile.mkdirs()
-    minioDockerfile.text = 
-            "FROM minio/minio:RELEASE.2019-01-23T23-18-58Z\n" +
-            "RUN mkdir -p /minio/data/${s3PermanentBucket}\n" +
-            "ENV MINIO_ACCESS_KEY ${s3PermanentAccessKey}\n" +
-            "ENV MINIO_SECRET_KEY ${s3PermanentSecretKey}"
+if (useFixture) {
+  apply plugin: 'elasticsearch.test.fixtures'
```

Comment:
Do we need to conditionally apply the plugin? What happens when we do but it's not applicable?

Reply:
We can't get into that situation because of the checks we have above this cod. Either you have all the necessary env variables set and run without Docker or you have non set and the Docker fixture gets started and used as an endpoint. Hope that makes sense?

Reply:
I think in other places we simply disable the task that uses the fixture rather than simply not applying the plugin, but I might be wrong here.

Reply:
I know that this was the only thing that worked when we created this code (it's just copied from the s3-repository plugin) and it took a lot of fiddling with Ryan and Alpar to get it right :D I'm happy to refactor this if possible but it might be best not to do that here :)

Reply:
Sounds good.